dotnet-efcore/EntityFrameworkCoreVsDapper/EntityFrameworkCoreVsDapper/Program.cs

```diff
@@ -1,15 +1,18 @@
-﻿using BenchmarkDotNet.Running;
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Jobs;
+using BenchmarkDotNet.Reports;
+using BenchmarkDotNet.Running;
+
 using EntityFrameworkCoreVsDapper.Repositories;
 
 public class Program
 {
     static void Main(string[] args)
     {
-        var efCoreRepository = new PersonsRepositoryEFCore();
-        efCoreRepository.GetXPersons(1000);
+        var personsRepository = new PersonsRepository();
+        personsRepository.GetXPersonsEFCore(1000);
 
-        var dapperRepository = new PersonsRepositoryDapper();
-        dapperRepository.GetXPersons(1000);
+        personsRepository.GetXPersonsDapper(1000);
```

Comment:
We aren't doing anything with these lines of code, so we could just remove them. Within the benchmark methods we are showing how to call the GetXPersons methods, so there isn't necessarily a need to also highlight it here.

Reply:
I need these lines for my article code to show how I can call these methods. Indeed when you see all the code it's clear how to call them, but looking at the code sections of the article it's not that clear.